docs/troubleshooting.md

```diff
@@ -0,0 +1,22 @@
+# How to troubleshoot issues
+
+Sometimes Ollama may not perform as you expect. One of the best ways to figure out what just happened is to take a look at the logs. You can find the logs on Mac by running the command:
+
+```shell
+cat ~/.ollama/logs/server.log
+```
+
+On Linux, the logs can be found with this command:
+
+```shell
+journalctl -u ollama
+```
+
+If you need help interpreting what you see, try signing in to the Discord at [discord.gg/ollama](https://discord.gg/ollama).
+
+## Error Index
```

Comment:
```suggestion
## Known issues
```

Reply:
Known issues to me means problems with the product we know about but need to fix, but these should be all the error messages that come up and what they mean.